frontend/src/components/Contexts/MissionDefinitionsContext.tsx

```diff
@@ -46,32 +32,9 @@ export const useMissionDefinitions = (): IMissionDefinitionsContext => {
 
     useEffect(() => {
         if (connectionReady) {
-            registerEvent(SignalREventLabels.missionDefinitionUpdated, (username: string, message: string) => {
-                const missionDefinition: CondensedMissionDefinition = JSON.parse(message)
-                missionDefinition.sourceType =
-                    Object.values(SourceType)[missionDefinition.sourceType as unknown as number]
-                setMissionDefinitions((oldMissionDefinitions) =>
-                    upsertMissionDefinition(oldMissionDefinitions, missionDefinition)
-                )
-            })
-            registerEvent(SignalREventLabels.missionDefinitionCreated, (username: string, message: string) => {
-                const missionDefinition: CondensedMissionDefinition = JSON.parse(message)
-                missionDefinition.sourceType =
-                    Object.values(SourceType)[missionDefinition.sourceType as unknown as number]
-                setMissionDefinitions((oldMissionDefinitions) =>
-                    upsertMissionDefinition(oldMissionDefinitions, missionDefinition)
-                )
-            })
-            registerEvent(SignalREventLabels.missionDefinitionDeleted, (username: string, message: string) => {
-                const mDef: CondensedMissionDefinition = JSON.parse(message)
-                if (!mDef.area) return
-                setMissionDefinitions((oldMissionDefs) => {
-                    const oldListCopy = [...oldMissionDefs]
-                    const queueIndex = oldListCopy.findIndex((m) => m.id === mDef.id)
-                    if (queueIndex !== -1) oldListCopy.splice(queueIndex, 1) // Remove deleted mission definition
-                    return oldListCopy
-                })
-            })
+            registerEvent(SignalREventLabels.missionDefinitionUpdated, (username: string, message: string) => {})
+            registerEvent(SignalREventLabels.missionDefinitionCreated, (username: string, message: string) => {})
+            registerEvent(SignalREventLabels.missionDefinitionDeleted, (username: string, message: string) => {})
```

Comment:
I'm not quite sure why we ignore these events in this commit? If we want to ignore them is it also not easiest to not register the event handler?